.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Deployment
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  CICD:
+    runs-on: ubuntu-latest
+    services:
+      docker:
+        image: docker
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.1
+    - name: Download and Install
+      uses: wei/curl@master
+      with:
+          args: curl -L https://raw.githubusercontent.com/datopian/ckan-cloud-helm/master/travis_ci_operator.sh > travis_ci_operator.sh
+    - name: Setup env variables
+      uses: FranzDiebold/github-env-vars-action@v2
+    - name: Install and init 
+      run: |
+        mkdir -p $HOME/bin/
+        mv travis_ci_operator.sh $HOME/bin/travis_ci_operator.sh
+        sudo bash $HOME/bin/travis_ci_operator.sh init
+        sudo bash .travis.sh install
+    - name: Run the script
+      run: |
+        sudo bash .travis.sh script
+    - name: Deploy
+      env:
+        TRAVIS_REPO_SLUG: MuhammadIsmailShahzad/ckan-cloud-helm
```

Comment:
@zelima just caught my own mistake this should be the link to datopian/ckan-cloud-helm
Fixing now

Reply:
:+1: eagle eye